SpywareFilter/sections/general_extensions.txt

```diff
@@ -403,17 +403,25 @@ accountkiller.com#%#window.ga = function(){var a = arguments[5];a&&a.hitCallback
 !
 ! Suppressing Google's "Protected" Audience API:
 ! https://wicg.github.io/turtledove/
-! TODO: Add a scriptlet: https://github.com/AdguardTeam/Scriptlets/issues/395
+! https://github.com/AdguardTeam/Scriptlets/issues/395
+#%#//scriptlet('no-protected-audience')
 !
+!#if (!ext_ublock)
 ! Suppressing both Google's Topics API and Topics API using Permissions policy
-! Works in CoreLibs-based AdGuard versions, the browser extension waits for https://github.com/AdguardTeam/tsurlfilter/issues/66
-! Part with "://*.*" before "$permissions=" is required, otherwise it will be marked as too wide rule
 ! AGLint temporarily disabled for this rule due to https://github.com/AdguardTeam/AGLint/issues/191
 ! TODO: Enable it back when AGLint is fixed.
-! TODO: remove $document modifier after versions with Corelibs 1.14.38+ will be released
-!#if (!ext_ublock)
 ! aglint-disable-next-line
-://*.*$document,permissions=join-ad-interest-group=()\,run-ad-auction=()\,browsing-topics=()
+*$permissions=join-ad-interest-group=()
```

Comment:
In the past such rules were marked as too wide, has it been fixed?
>! Error: "The rule is too wide, add domain restriction or make the pattern more specific" in the rule: `"*.*$permissions=join-ad-interest-group=()\, run-ad-auction=()\, browsing-topics=()"`
`*.*$permissions=join-ad-interest-group=()\, run-ad-auction=()\, browsing-topics=()`

https://github.com/AdguardTeam/FiltersRegistry/blob/e529458be21800609358ce99ff1bcc8bb5b02305/filters/filter_3_Spyware/diff.txt

Reply:
Extension and app show them as applied)
Which blocker has this isssue?

Reply:
As I recall, it wasn't a specified blocker but it was rejected in filters registry.

Reply:
It was just a question, maybe it has been fixed already and `*$permissions=` can be used.